man.Rmd

```diff
@@ -505,33 +516,38 @@ Here's an example from `tidyr::chop()`:
 #' df <- tibble(x = 1:4, y = list(integer(), 1L, 1:2, 1:3))
 #' df %>% unchop(y)
 #' df %>% unchop(y, keep_empty = TRUE)
-#' 
-#' #' # Incompatible types -------------------------------------------------
+#'
+#' # Incompatible types ================================================
```

Comment:
Jenny TODO: sync up with whatever comes of https://github.com/tidyverse/tidyr/pull/1412